uaa/src/main/resources/log4j2-docker.properties

```diff
@@ -0,0 +1,30 @@
+status = error
```

Comment:
Is this a new addition from upstream? I wasn't able to find it in our version for 4.30. I did see a similar properties file is available in upstream UAA, but the folder location is different. Are these two supposed to be the same?

https://github.com/cloudfoundry/uaa/blob/develop/scripts/cargo/log4j2.properties

Reply:
because of tomcat:9.0.44-jdk11-openjdk-slim and upstream UAA code requires a log4j2.properties, but can't take from jvm opts -D from cli. see the comments and code here : https://github.com/cloudfoundry/uaa/blob/develop/server/src/main/java/org/cloudfoundry/identity/uaa/impl/config/YamlServletProfileInitializer.java#L185-L199
when the docker image starts, it can't access this  scripts/cargo/log4j2.properties, I copied the file content to this location and rename it. it is possible to have different content.